web/client/utils/StateUtils.js

```diff
@@ -203,26 +286,10 @@ export const updateStore = ({ rootReducer, rootEpic, reducers = {}, epics = {} }
  * @param {object} store the store to update, if not specified, the persisted one will be used
  */
 export const augmentStore = ({ reducers = {}, epics = {} } = {}, store) => {
-    const rootReducer = fetchReducer();
-    const reducer = persistReducer((state, action) => {
-        const initialStoreKeys = Object.keys(rootReducer({}, {}));
-        const newState = {...state, ...rootReducer(state, action)};
-        return Object.keys(reducers)
-            // avoid to update the state twice
-            // if the original store contains the same reducers
-            .filter((key) => initialStoreKeys.indexOf(key) === -1)
-            .reduce((previous, current) => {
-                return {
-                    ...previous,
-                    [current]: reducers[current](previous[current], action)
-                };
-            }, newState);
-    });
-    (store || getStore()).replaceReducer(reducer);
-
-    const rootEpic = fetchEpic();
-
-    wrapEpics(epics).forEach((epic) => {
-        rootEpic.next(epic);
+    const persistedStore = store || getStore();
+    Object.keys(reducers).forEach((key) => {
+        persistedStore.storeManager.addReducer(key, reducers[key]);
     });
+    persistedStore.dispatch({type: 'REDUCERS_LOADED'});
+    persistedStore.storeManager.addEpics('notMutable', wrapEpics(epics));
```

Comment:
It is not clear how this `notMutable` key works to me.
Is it working because there is not a plugin called notMutable so the mute and unmute handler will not be triggered?

Reply:
Exactly, epics are mutable by "notMutable" key, and this will never happen because there is no such plugin.
But augmentStore was used only by withExtensions HOC. With the recent commits this HOC was slightly updated and now it uses storeManager.addEpics & storeManager.addReducer methods to manager reducers and epics.